### PR TITLE
QRNG banner

### PR DESCRIPTION
**Pull request by wkande**
A simple deprecated banner on pages belonging to QRNG content.

### CODE REVIEW

File: docs/guides/qrng/qrng-remix/index.md [hunk 1]
```diff
@@ -11,12 +11,16 @@ tags:
...
 
+<QrngRemoval/>
+
 <SearchHighlight/>
 
 <FlexStartTag/>
 
 # {{$frontmatter.title}}
 
+<QrngRemoval/>
```
Comment (dcroote) [inline docs/guides/qrng/qrng-remix/index.md:22]: Shouldn't be on this page twice...

File: docs/_components/QrngRemoval.vue [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<template>
+  <div
+    style="
```
Comment (dcroote) [inline docs/_components/QrngRemoval.vue:3]: Maybe it should stand out just a little more by using a different background? For example like `::: warning`?

File: docs/guides/qrng/qrng-remix/index.md (hunk 1 above)
Comment (wkande) [reply docs/guides/qrng/qrng-remix/index.md:22]: Thanks

File: docs/_components/QrngRemoval.vue (hunk 2 above)
Comment (wkande) [reply docs/_components/QrngRemoval.vue:3]: I'm going off a figma given to me but I will suggest it.

Comment (dcroote) [reply docs/_components/QrngRemoval.vue:3]: It was just a thought - not worth spending a lot of time on

**Review by dcroote — APPROVED** [review]
Seems good to go

**Pull request merged**